web/front/app.html

```diff
@@ -1,4 +1,7 @@
 <div>
+  <v-alert :value="showDrilldownAlert" type="info" class="drilldownAlert">
```

Comment:
if the user has chosen to use the drill functionality, why do they need an alert?

Reply:
generally you have to be careful with these as they require the user to click to close them, which is extra work for the user - or does this one not need that?